src/Agent/NewRelic/Agent/Core/Transactions/Transaction.cs

```diff
@@ -892,11 +892,6 @@ public void SetReferrerUri(string uri)
 
         public void SetQueueTime(TimeSpan queueTime)
         {
-            if (queueTime == null)
```

Comment:
We're sure this can't be null?

Reply:
Yep. `TimeSpan` is a struct and can never be null. The old code compiled because `TimeSpan` has an `operator ==()` overload. But trying to set a `TimeSpan` to `null` won't compile. 
![image](https://github.com/newrelic/newrelic-dotnet-agent/assets/120425148/da5b99f7-6599-465f-b185-0152100f68ef)